relay-server/src/services/store.rs

```diff
@@ -599,6 +599,15 @@ impl StoreService {
                     event_type = message.variant(),
                     topic = topic_name,
                 );
+                if topic_name == "ingest-feedback-events" {
+                    if let KafkaMessage::Event(event_msg) = message {
+                        relay_log::info!("Producing to ingest-feedback-events.\n start_time={}\n event_id={}\n project_id={}",
```

Comment:
Is there a way to pass a kv dict of "extras" like in python?

Reply:
Why are our existing metrics not enough?

We shouldn't log in potentially high throughput code (on info level).

Reply:
Ha, nvm, I just read your description. I'll take a look at the metrics tomorrow.

Reply:
Sounds good! This log won't be high throughput. Since the topic, consumer, and rollout rate only exist/are set in s4s. And there's little to no feedback sent in s4s, except employees who use the feedback widget on a few sentry pages (ex: performance).